PeerioTechnologies/peerio-desktop
# fix(NewChannel): keep button disabled when over room limits

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/3263/desktop-paywall-feedback-in-new-room-invite-branch
 
#### Testing instructions  
When user has reached room limit and attempts to create a new room, the "Open" button to create a new room should remain disabled.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
